index.html

```diff
@@ -258,6 +258,14 @@ <h3>The <dfn><code>VisualViewport</code></dfn> interface</h3>
                 Although it is referred to as the pinch-zoom scaling factor, it can be affected through means other than pinch-zooming. e.g. When the UA centers and zooms in a focused edit box.
               </div>
             </dd>
+            <dt><dfn>segments</dfn></dt>
+            <dd>
+              Returns an array of DOMRects that represent the dimensions of each existing viewport segment. Returns null when called from within an iframe context. 
```

Comment:
We should add a sentence that `segments` is null when there is only a single segment (i.e. the base case) to avoid redundancy, confustion, and to ease in detecting when the information might be relative (wouldn't want someone to start coding to visualViewport.segments[0] since that'll fall down when moving into a spanned mode).